dashboard/config/locales/slides.en.yml

```diff
@@ -2020,3 +2020,204 @@ en:
       8:
         image: 'notes/mc_2017_congratulations_8.png'
         text: "Stampy: How did you do that? Stacy: It was easy, actually. I just used code!"
+    csd_concept_how_computers_work:
+      1:
+        image: 'notes/csd_concept_how_computers_work_1.png'
+        text: "Everywhere you look, computers are changing the world. Whether they're on our desktops, in our homes, our pockets, or just about anywhere else. But while most of us use this revolutionary technology daily, we don't often ask: \"How do computers work?\" In this series, you're going to learn how computers really work, starting with what makes a computer a computer. Then, you'll look at how information is represented inside the computer using tiny electrical signals going on and off at mind-boggling speeds."
+      2:
+        image: 'notes/csd_concept_how_computers_work_2.png'
+        text: "After that, you'll learn how computers use circuits to do everything from simple math to simulating entire virtual worlds. Next, you'll zoom out and look at the different parts of a computer that actually input, output, and store all that information. Finally, you'll take a look at what code actually is and how software controls hardware. So whether you're just curious about the devices you use every day or you want to design the innovations of the future, the first step is learning how computers work."  
+    csd_concept_computer:
+      1:
+        image: 'notes/csd_concept_computer_1.png'
+        text: "My name is May-Li Khoe and I'm a designer and an inventor. So some of the things I've designed have been at Apple, and now I design products for kids to use so that they can have an easier time in school. My other jobs include DJ-ing and dancing."
```

Comment:
is this correct name spelling?

Reply:
Yup! [Linked In](https://www.linkedin.com/in/maylikhoe/)